templates/registration/contact_thanks.html

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
     <h1>Contact</h1>
-    <div class="contact-thanks">
+        <div class="contact-thanks">
     <p>Thank you! Your message was received successfully.</p>
```

Comment:
Oh, haha. Nope. The plus sign comes from Git- it's just showing that the line was added.